README.md

```diff
@@ -12,20 +16,14 @@ an exact match.
 
 There are also generic versions of the functions for non-string inputs.
 
-## Installation
+## Usage
 
-`strsim` is available on [crates.io](https://crates.io/crates/strsim). Add it to
-your `Cargo.toml`:
+Add this to your `Cargo.toml`:
 ```toml
 [dependencies]
-strsim = "0.9.2"
+strsim = "0.9"
 ```
 
-## Usage
-
-Go to [Docs.rs](https://docs.rs/strsim/) for the full documentation. You can
```

Comment:
I'd like to leave this line in there for people who want to see the function signatures. I know you added the docs.rs badge (thanks for that!), but that's easy to miss.